### PR TITLE
Add Governance doc and update members files

### PR DESCRIPTION
**Pull request by killianmuldoon**
Add a Governance doc which:
1) Defines organisation roles.
2) Defines a simple process for adding and removing members and people from project roles.
3) Defines a simple process for adding and removing projects from project roles.

Update the OWNERS and MEMBERS files with the actual state.

As discussed in the [February 22nd NPWG community meeting](https://docs.google.com/document/d/1oE93V3SgOGWJ4O1zeD1UmpeToa0ZiiO6LqRAmZBPFWM/edit#heading=h.5cn4jef8loy1)

### CODE REVIEW

**Review by killianmuldoon — COMMENTED** [review]
@dougbtv @adrianchiris @s1061123 @moshe010 

Not sure who else to bring in on this one - but would be good to understand if I'm going in the right direction. 

I mostly tried to reflect the current process and add a layer of PRs so that we have a source of truth and a changelog.

**Comment by plwhite** [comment]
This looks great, many thanks for doing all this. Happy to merge, but going to sit on it for a bit so other people can comment too.

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+## K8sNetworkPlumbingWG Governance
```
Comment (adrianchiris) [inline GOVERNANCE.md:1]: i assume this adds on to[1] ?
perhaps (until its merged into one) we should mention it here.


[1] https://docs.google.com/document/d/1lIWOK-W6fb1VZiSjO1BoFUs0TmoVwgd4XuVc6mtJk2c/edit#heading=h.5x0d5h95i329

Comment (killianmuldoon) [reply GOVERNANCE.md:1]: I didn't even realise that doc existed 😆. I think we should try to merge this PR as is - I'll mention the other doc in it. I'll do a follow-up round where I add the actual info from that doc and we can deprecate the Google Doc version eventually.

Thanks for pointing that out!

**Comment by killianmuldoon** [comment]
Updated to make this doc an evolution of the original proposal on governance.

File: GOVERNANCE.md (hunk 1 above)
Comment (killianmuldoon) [reply GOVERNANCE.md]: I've updated this to make it an amendment to the existing doc - thanks for pointing to that.

**Review by dougbtv — APPROVED** [review]
Thank you!

**Pull request merged**